sky/clouds/nebius.py

```diff
@@ -20,6 +22,28 @@
     nebius.NEBIUS_CREDENTIALS_FILENAME
 ]
 
+_INDENT_PREFIX = '    '
+
+
+def nebius_profile_in_aws_cred() -> bool:
+    """Checks if Nebius Object Storage profile is set in aws credentials
+     and has endpoint_url."""
+
+    profile_path = os.path.expanduser('~/.aws/credentials')
+    nebius_endpoint_re = re.compile(r'/^endpoint_url *= *https:\/\/storage\.[a-zA-Z1-9\-]*\.nebius\.cloud:443$/gm')  # pylint: disable=line-too-long
+    nebius_profile_exists = False
+    nebius_endpoint_exists = False
+    if os.path.isfile(profile_path):
+        with open(profile_path, 'r', encoding='utf-8') as file:
+            for line in file:
+                if f'[{nebius.NEBIUS_PROFILE_NAME}]' in line:
+                    nebius_profile_exists = True
+                if nebius_endpoint_re.match(line):
+                    nebius_endpoint_exists = True
+                    break
```

Comment:
```suggestion
    nebius_profile_exists = False
    nebius_endpoint_exists = False
    profile_path = os.path.expanduser('~/.aws/credentials')
    if os.path.isfile(profile_path):
        with open(profile_path, 'r', encoding='utf-8') as file:
            for line in file:
                if f'[{nebius.NEBIUS_PROFILE_NAME}]' in line:
                    nebius_profile_exists = True
                if nebius_endpoint_re.match(line):
                    nebius_endpoint_exists = True
                    break
    nebius_endpoint_re = re.compile(r'/^endpoint_url *= *https:\/\/storage\.[a-zA-Z1-9\-]*\.nebius\.cloud:443$/gm')  # pylint: disable=line-too-long
    config_path = os.path.expanduser('~/.aws/config')
    if os.path.isfile(config_path):
        with open(config_path, 'r', encoding='utf-8') as file:
            for line in file:
                if nebius_endpoint_re.match(line):
                    nebius_endpoint_exists = True
                    break
```
we may need to do something like this, since `endpoint_url` appears in `~/.aws/config` and not `~/.aws/credentials`

Reply:
(resolving as I no longer believe this is relevant / needed: see thread above)